pkgs/applications/networking/cluster/k3s/docs/CLUSTER_UPKEEP.md

```diff
@@ -0,0 +1,86 @@
+
+# K3s Upkeep for Users
+
+General documentation for the K3s user for cluster tasks and troubleshooting steps.
+
+## Upkeep
+
+### Changing K3s Token
+
+Changing the K3s token requires resetting cluster. To reset the cluster, you must do the following:
+
+#### Stopping K3s
+
+Disabling K3s NixOS module won't stop K3s related dependencies, such as containerd or networking. For stopping everything, either run "k3s-killall.sh" script or reboot host.
```

Comment:
Could you please explain how to run k3s-killall.sh on nixos? It was packaged on https://github.com/NixOS/nixpkgs/pull/309099.

Reply:
Its available on default path, added a note to that effect to make it explicit. There are no arguments for the script and is just run directly.